packages/qdrant-loader/src/qdrant_loader/core/qdrant_manager.py

```
@@ -215,30 +215,35 @@ def create_collection(self) -> None:
-            except Exception:
-                vector_size = None
+                vs = self.settings.llm_settings.embeddings.vector_size
+            except AttributeError:
+                vs = None
+            if vs is not None:
+                try:
+                    vector_size = int(vs)
+                except (TypeError, ValueError) as exc:
+                    raise ValueError(
+                        "Invalid global.llm.embeddings.vector_size; expected an integer"
+                    ) from exc
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)